filters/ema.cpp

```diff
@@ -0,0 +1,25 @@
+#include "filters/ema.hpp"
+
+namespace mesos {
+namespace serenity {
+
+IpcEMAFilter::~IpcEMAFilter() {}
+
+
+Try<Nothing> IpcEMAFilter::consume(ResourceUsage_Executor& in)
```

Comment:
We can do that as 1 option. 

As 2 option we can do 
```
template<typename T, typename S>
class MultiConsumer: public Consumer<T>, public Consumer<S>
{
...
virtual Try<Nothing> consume(T& in) = 0;
virtual Try<Nothing> consume(S& in) = 0;
};
```
We can discuss which is preferable.